src/app/DemoScreen.tsx

```diff
@@ -19,17 +22,41 @@ const mapStateToProps = (state: RootState) => ({
 type Props = ReturnType<typeof mapStateToProps>;
 
 const DemoScreen = IsLoggedIn(({ profile }: Props) => (
-  <div>
+  <AdminLayout
+    logo={<Logo />}
+    headerContent={
+      <div style={{ flex: 1, textAlign: 'right', paddingRight: 22 }}>
+        <Link to={AUTH_ROUTER_PATHS.logout}>Logout</Link>
+      </div>
+    }
+  >
     {profile && (
       <>
-        ID: {profile.id}
-        <br />
-        Email: {profile.email}
-        <br />
+        <p>ID: {profile.id}</p>
+        <p>Email: {profile.email}</p>
       </>
     )}
-    <Link to={AUTH_ROUTER_PATHS.logout}>Logout</Link>
-  </div>
+  </AdminLayout>
 ));
 
 export default connect(mapStateToProps)(DemoScreen);
+
+const Logo = () => {
+  const { sidebarState } = useContext(AdminLayoutContext);
+
+  return (
+    <h1
+      style={{
+        color: sidebarState === SidebarState.CLOSED_DRAWER ? 'black' : 'white',
```

Comment:
Context could provide `isSidebarClosed` so you don't have to use the SidebarState enum